sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Triggers/ServiceBusTriggerAttributeBindingProvider.cs

```diff
@@ -112,5 +120,40 @@ private bool GetAutoCompleteMessagesOptionToUse(ServiceBusTriggerAttribute attri
 
             return _options.AutoCompleteMessages;
         }
+
+        internal static void LogExceptionReceivedEvent(ProcessErrorEventArgs e, string functionName, ILoggerFactory loggerFactory)
+        {
+            try
+            {
+                var errorSource = e.ErrorSource;
+                var logger = loggerFactory?.CreateLogger(LogCategories.CreateFunctionCategory(functionName));
```

Comment:
Won't this end up creating a new logger for every error?

Reply:
I guess it was already like this, but would we be able to pass in a specific logger to this method?

Reply:
yes, this can be done.